egs/wsj/s5/utils/validate_dict_dir.pl

```diff
@@ -68,6 +75,11 @@
   } elsif (!$silence{$col[0]}) {
     set_to_fail(); print "--> ERROR: phone $col[0] not found in $dict/silence_phones.txt\n";
   }
+  if ($crlf == 1 && m/\r$/) {
```

Comment:
no need for $ at the end of these expressions... \r anywhere in the line is bad [although less likely].